src/common/core/constants.js

```diff
@@ -39,6 +39,17 @@ define([], function () {
         META_POINTER_PREFIX: '_p_',
         META_ASPECT_PREFIX: '_a_',
 
+        MIXINS_SET:'_mixins',
+        MIXIN_ERROR_TYPE:{
+            MISSING: 'missing',
+            ATTR_COLLISION:'attribute collision',
```

Comment:
Style: spaces.
Do NOT use short variable names.
CONT vs CONST hard to read. Spell all of them out.